bin/deploy.sh

```diff
@@ -19,6 +19,9 @@ success_msg "Correct Config File"
 # build contracts if needed
 build_contracts "./.."
 
+# create deployment folder
+mkdir $BIN_DIR/../deployments
```

Comment:
let's check if the directory exists
```suggestion
[ ! -d "$BIN_DIR/../deployments" ] && mkdir $BIN_DIR/../deployments
```

Reply:
Even simpler:
```suggestion
mkdir -p $BIN_DIR/../deployments
```